app/models/pano/email_validator.rb

```diff
@@ -1,11 +1,12 @@
 module Pano
   class EmailValidator < ActiveModel::EachValidator
-    EMAIL_REGEX = /\A[a-z0-9][a-z0-9._%+-]*(?<!\.)@(?:[-a-z0-9]+\.)+[a-z]{2,}\z/i
+    # https://emailregex.com
+    EMAIL_REGEX = /(?:[a-z0-9!#$%&'*+\/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+\/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/i
```

Comment:
Needs a \A at the start and a \Z at the end, or else it will match only on part of the given string. E.g.,
`dave isaacs@gmail.com com` will match, because `isaacs@gmail.com` matches.